### ISSUE TITLE
May 30, 2015 Utah ng-bridge

### ISSUE THREAD

**Issue by ppkn** [ng-bridge/organizing#2]
## Utah ng-bridge May 30, 2015

I'm pretty much just following @judytuna's lead... Let's do this!
This workshop's going to be amazing!! Here's what we need to do!

A few months out:
- [ ] Find a venue and sponsor for the workshop. Sponsoring the workshop costs between $400 and $900, depending on the size of the workshop.
- [x] Find two co-organizers. It's a lot of work, but always a lot of fun. No experience is necessary; checkout the [workshop cookbook](https://github.com/railsbridge/docs/wiki/Cookbook) if it's new to you. (Also, we match up first time organizers with mentors.)

Leave a comment below if you're able to host, sponsor or organize! :smile_cat: 

Once we have a a venue, sponsors, and organizers, we can:
- [ ] Work together to set an RSVP limit that works for everyone
- [ ] Post the workshop to [Bridge Troll](http://www.bridgetroll.org/)
- [ ] Post the workshop to appropriate Meetup group(s) for publicity 

One to two weeks out, we can:
- [ ] Plan a teacher training
- [ ] Order breakfast
- [ ] Order lunch
- [ ] Find a good place for the afterparty

After the workshop, we can:
- [ ] Party! :tada:  
- [ ] Retro
- [ ] Post-workshop survey

:panda_face:

**Comment by ppkn** [comment]
I can be a co-organizer.

**Comment by ppkn** [comment]
@iheartrachie, want to be involved in this?

**Comment by ppkn** [comment]
I set the date to be May 30th. I also got Jessica Olsen involved and I'm encouraging her to actually use GitHub :octocat: :smile:

**Comment by rachelpipkin** [comment]
Sorry! I never got a notification about this message!

I'd love to help. Have you talked with GDI at all? Where would be an ideal location for a venue? My initial thoughts are south salt lake valley like South Jordan/Draper but Thanksgiving point area may work as well. We'll probably want to reach out to companies that are Angular shops as I think they'll have a lot more interest in the event. Let's brainstorm a list. Maybe HireVue?

**Comment by ppkn** [comment]
Domo, STG, Workfront, Pluralsight, and InsideSales (those were all sponsors
at ng-conf). Adobe also hosts an Angular meetup every month so they might
be interested. Jessica recommended Salt Lake Valley as well. I'm not sure
how many people might come so the location is up in the air. Also, what
would you think about presenting?

**Comment by rachelpipkin** [comment]
Those are all great places to look at. I'm not really a presenter but I can refer you to some people. Do you know how long we'll need to fill? What subjects are you looking for?

**Comment by ppkn** [comment]
I imagine it being an all day workshop; so what would that be... like noon-7? It will mostly be part of a "working app" tutorial, but it might be nice to have a presenter or two in there to talk about things like applying for jobs, other educational resources, open source projects, etc.

**Comment by ppkn** [comment]
Since there hasn't been a lot of support in the community and I've been busy at work, I don't know if this is going to come together by the 30th...

**Comment by rachelpipkin** [comment]
Yeah this summer is getting crazy busy and I won't be around for most of it. Let's not give up. I think this is a really good idea. There was a strong Angular presence at OpenWest so I think there would be a lot of interest. What about trying for something in the fall? Also, I think Jessica and I would be more useful with a different communication channel. Neither of us frequent GitHub enough.
